pytorch_lightning/tuner/tuning.py

```diff
@@ -45,14 +52,37 @@ def _tune(
         if self.trainer.auto_scale_batch_size:
             if isinstance(self.trainer.auto_scale_batch_size, str):
                 scale_batch_size_kwargs.setdefault("mode", self.trainer.auto_scale_batch_size)
-            result["scale_batch_size"] = scale_batch_size(self.trainer, model, **scale_batch_size_kwargs)
```

Comment:
Worth to share the value would be overridden. IMO, the provided one should override the trainer one.

Reply:
sorry, didn't get you here.